index.md

```diff
@@ -1,12 +1,17 @@
 # Anbox Cloud documentation
 
-Anbox Cloud enables running Android apps on any cloud platform at scale. It uses system containers or virtual machines to run the nested Android containers and [Juju](https://juju.is/) for deployment in a cloud environment.
 
-Anbox Cloud supports x86 and ARM64 hardware, providing the same set of features for both architectures.
+Anbox Cloud runs Android in the cloud using lightweight LXD containers, offering VM-level isolation without the associated overhead. Built on Ubuntu, it provides a scalable platform to deploy, manage, and stream Android workloads across public or private infrastructure. Compared to other Android emulation solutions, Anbox Cloud can provide at least twice the density and can serve up to **100 Android instances per server**.
```

Comment:
We don't just offer containerized solutions, our VM images are also worth mentioning.